test/testcafe/spec/ChallengeCustomOTPAuthenticatorView_spec.js

```diff
@@ -67,13 +69,12 @@ test.requestHooks(mockInvalidPasscode)('challege custom otp authenticator with i
   const challengeOnPremPage = await setup(t);
   await checkA11y(t);
   await challengeOnPremPage.verifyFactor('credentials.passcode', 'test');
-  await challengeOnPremPage.clickNextButton();
-
-  await t.expect(challengeOnPremPage.getInvalidOTPError())
-    .eql('Invalid code. Try again.');
+  await challengeOnPremPage.clickNextButton('Verify');
+  await t.expect(challengeOnPremPage.getInvalidOTPError()).eql('Invalid code. Try again.');
 });
 
-test.requestHooks(mockChallengeAuthenticatorCustomOTP)('should show custom factor page link', async t => {
+// OKTA-465319 Help link is not supported in v3
+test.meta('v3', false).requestHooks(mockChallengeAuthenticatorCustomOTP)('should show custom factor page link', async t => {
```

Comment:
this will be able to be enabled once Denys PR is merged: https://github.com/okta/okta-signin-widget/pull/3236